### PR TITLE
feat: add `n` flag to `Xperf`

### PR DESCRIPTION
**Pull request by magnus-madsen**

### CODE REVIEW

File: main/src/ca/uwaterloo/flix/Main.scala [hunk 1]
```diff
@@ -376,7 +378,11 @@ object Main {
             .required()
         )
 
-      cmd("Xperf").action((_, c) => c.copy(command = Command.CompilerPerf)).hidden()
+      cmd("Xperf").action((_, c) => c.copy(command = Command.CompilerPerf)).children(
+        opt[Int]("n")
```
Comment (mlutze) [inline main/src/ca/uwaterloo/flix/Main.scala:382]: Does the command line parser have a "default" option that would automatically display in the usage text?

**Review by mlutze — APPROVED** [review]

File: main/src/ca/uwaterloo/flix/Main.scala (hunk 1 above)
Comment (magnus-madsen) [reply main/src/ca/uwaterloo/flix/Main.scala:382]: Huh?

Comment (mlutze) [reply main/src/ca/uwaterloo/flix/Main.scala:382]: E.g. we write `cmd(....).default(CompilerPerf.Default)`

And then when the user does `flix --help` it displays

```
usage: blah blah
    -Xperf n       default: 7
```

Comment (magnus-madsen) [reply main/src/ca/uwaterloo/flix/Main.scala:382]: Not sure, but in any case, the plan is to hide all `X` options and this one is already hidden.

**Pull request merged**